rustfmt.toml

```diff
@@ -0,0 +1,102 @@
+# title: Ix Standard Rustfmt Configuration
+# author: John C. Burnham <john@argument.xyz>
+
+# For additional information about Rustfmt options see:
+# https://rust-lang.github.io/rustfmt/?version=v1.8.0
+#
+# General
+#version              = "One"              # rustfmt 2.0 is not released yet
```

Comment:
This can be removed, as `version` has been deprecated in favor of the unstable `style_edition` per https://rust-lang.github.io/rfcs/3338-style-evolution.html, which controls the edition of the Rust style guide used for formatting.

The RFC notes that:
> By default, rustfmt and cargo fmt will use the same edition for style as the Rust edition used for parsing.

This means that unless we want to separately commit changes to the Rust edition and style guide, which I don't think is necessary, we can let the `edition` control the parser as well as the style spec.

Reply:
It's already commented out